lib/tasks/organization_images.rake

```diff
@@ -0,0 +1,8 @@
+namespace :organization_images do
+  desc "Backfill organization images"
+  task backfill: :environment do
+    ActiveStorage::Attachment.where(record_type: "User", name: "images").each do |asa|
+      asa.update(record_type: "Organization", record_id: asa.record.organizations.first.id)
```

Comment:
Are there any cases where `record.organizations.first` may be nil?

Reply:
There aren't - checked with a clean dump